frontend/src/components/EditAdminTeam.js

```diff
@@ -72,83 +72,88 @@ export default function EditAdminTeam({ user }) {
-									label={searchedUserIsInAdminTeam
+				{
+					requestSentIfUserIsInAdminTeam
+					&&
+					<Box component="form" noValidate onSubmit={updateTeamsOfUser} sx={{ mt: 1 }} style={{ width: "100%" }}>
+						<Divider sx={{ mb: 2 }} />
+						<Grid container justify="space-between" style={{ width: "100%" }}>
+							<>
+								<ParagraphTypography variant="body1" align="left" gutterBottom sx={{ mb: 2 }} >
+									{searchedUserIsInAdminTeam ?
+										<>The user with the email &quot;<b>{searchedUserEmail}</b>&quot; is <b>in</b> the admin team. Below, you can remove them from the admin team.</>
+										:
+										<>The user with the email &quot;<b>{searchedUserEmail}</b>&quot; is <b>not</b> in the admin team. You can invite them to the admin team below.</>
+									}
+								</ParagraphTypography>
+								<FormGroup>
+									<FormControlLabel
+										label={searchedUserIsInAdminTeam
+											?
+											<ParagraphTypography>Are you sure you want to <b>remove</b> this user from the admins team?</ParagraphTypography>
+											:
+											<ParagraphTypography>Are you sure you want to <b>invite</b> this user to the admins team?</ParagraphTypography>
+										}
+										control={<Checkbox checked={areYouSureChecked} onChange={() => setAreYouSureChecked(prevState => !prevState)}/>}
+									/>
+								</FormGroup>
+								<Button
+									type="submit"
+									disabled={!areYouSureChecked}
+									fullWidth
+									variant="contained"
+									sx={{ mt: 3, mb: 2 }}
+								>
+									{searchedUserIsInAdminTeam
 										?
-										<ParagraphTypography>Are you sure you want to <b>remove</b> this user from the admins team?</ParagraphTypography>
+										<>Remove user from admin team</>
 										:
-										<ParagraphTypography>Are you sure you want to <b>invite</b> this user to the admins team?</ParagraphTypography>
+										<>Send invite</>
 									}
-									control={<Checkbox checked={areYouSureChecked} onChange={() => setAreYouSureChecked(prevState => !prevState)}/>}
-								/>
-							</FormGroup>
-							<Button
-								type="submit"
-								disabled={!areYouSureChecked}
-								fullWidth
-								variant="contained"
-								sx={{ mt: 3, mb: 2 }}>
-								{searchedUserIsInAdminTeam
-									?
-									<>Remove user from admin team</>
-									:
-									<>Send invite</>
-								}
-							</Button>
-							{successMessage !== null && <Grid item xs={12}><Alert severity="info">{successMessage}</Alert></Grid>}
-							{errorBottomMessage !== "" && <Grid item xs={12}><Alert severity="error">{errorBottomMessage}</Alert></Grid>}
-						</>
+								</Button>
+								<ConditionalAlert severity="info" text={successMessage} conditionFunction={(inputElement)=> inputElement !== null}/>
+								<ConditionalAlert severity="error" text={errorBottomMessage} />
+							</>
 
-					</Grid>
-				</Box>
-			}
+						</Grid>
+					</Box>
+				}
+			</Grid>
 		</Grid>
-	</Grid>;
+	</>;
```

Comment:
Will reduce the LOC in my next PR.

Reply:
Lines of Code ;-)  is no good readability metric. Typically, some more lines of code are better readable. It's about levels of detail and about complexity (number of features or size of features). The complexity also increases with density of code. For low-level operations it's shorter to write bit magic but it's very bad readable. More levels of detail are easier to oversee and review because with more levels it's not all thrown together but separate ideas are separated. Function and variable names can be used as concise code explanations. And you already did that :-) .

I know, as (technical) students, we don't have the years of software engineering and human experience to write code in a way which is easy to review, reuse, extend and read for humans. In ideal theory, code is written once and doesn't change later but only is extended (Open-Closed Principle). This also keeps git-diffs small. There isn't only one way to craft those properties but multiple levels of details work for most of the people (except for the Assembler-kind programmers).

This often just becomes apparent if one needs to fight with hundreths of thousands of lines of code.